src/extension/sqlserver/src/main/resources/applicationContext.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 
+ Copyright (C) 2021 - Open Source Geospatial Foundation. All rights reserved.
+ This code is licensed under the GPL 2.0 license, available at the root
+ application directory.
+ -->
+<!DOCTYPE beans PUBLIC "-//SPRING//DTD BEAN//EN" "http://www.springframework.org/dtd/spring-beans.dtd">
+
+<beans>  
+  <bean id="sqlServerExtension" class="org.geoserver.platform.ModuleStatusImpl">
```

Comment:
@jodygarnett - is this what I should be using or is there (or should there be) an implementation for JDCB stores to check if driver is available?